ci/fih_test_docker/execute_test.sh

```diff
@@ -34,6 +34,9 @@ else
 fi
 
 # build TF-M with MCUBoot
+
+export GIT_SSL_NO_VERIFY=1
```

Comment:
What's the purpose of this? I'd be concerned about disabling SSL

Reply:
When I tried to run the scripts in my local environment I faced the issue below during TF-M build. This is a workaround for that:

```
Cloning into 'tfm_test_repo-src'...
fatal: unable to access 'https://git.trustedfirmware.org/TF-M/tf-m-tests.git/': server certificate verification failed. CAfile: none CRLfile: none

Interestingly I can pull the code from the same machine, but the scripts run in a docker container. Might this cause the issue?

Reply:
Removed since it not an issue in GitHub CI.